### ISSUE TITLE
Proper Motion not applied correctly to stars

### ISSUE THREAD

**Issue by wrb1** [brandon-rhodes/pyephem#154]
I believe the star data is from Hipparcos, but to quote from that site
https://heasarc.gsfc.nasa.gov/W3Browse/all/hipparcos.html

Right ascension in the specified equinox for epoch J1991.25. This was given in the ICRS reference system (J2000 equator)... 

Ditto for declination.

The problem is the proper motion in PyEphem is applied assuming the positions are as of 2000, rather than 1991.25.  You can see the discrepancy with a "fast" moving star e.g. Arcturus.

**Comment by brandon-rhodes** [comment]
Thanks for the report! Could you provide a small program that prints a coordinate that looks incorrect, so that we can create a test that would define the correct behavior PyEphem should have? Thanks!

**Comment by wrb1** [comment]
Compare the output of ephem for
s= ephem.star('Arcturus')
s.compute(ephem.date("2017/1/1 12:00"))
s.ra
s.dec

with
http://aa.usno.navy.mil/cgi-bin/aa_geocentric.pl?ID=AA&task=6&body=-6&year=2017&month=1&day=1&hr=12&min=0&sec=0.0&intv_mag=1.0&intv_unit=1&reps=5

                             Arcturus                              
     
                   Apparent Geocentric Positions                   
                 True Equator and Equinox of Date                  
     
   Date        Time           Right Ascension          Declination
        (UT1)  
                       h  m   s                h  m   s                      °    '      "
2017 Jan 01 12:00:00.0          14 16 25.115          + 19 05 39.64
2017 Jan 02 12:00:00.0          14 16 25.146          + 19 05 39.39
2017 Jan 03 12:00:00.0          14 16 25.175          + 19 05 39.15
2017 Jan 04 12:00:00.0          14 16 25.203          + 19 05 38.93
2017 Jan 05 12:00:00.0          14 16 25.230          + 19 05 38.73

**Comment by brandon-rhodes** [comment]
Those do look like Hipparcos coordinates in `ephem/stars.py`, don't they?

```
Arcturus,f|S|K2,14:15:40.3|-1093.45,19:11:14|-1999.4,-0.05,2000,0
```

So I switched the "2000" to "1991" and re-ran your suggested script. Before it said:

```
14:16:25.75
19:05:57.1
```

With "1991" it now says:

```
14:16:50.39
19:03:09.9
```

Which has moved the position farther away from the USNO result, rather than closer? And the effect appears to be much larger than the difference we want to eliminate?

**Comment by wrb1** [comment]
I did not suggest changing 2000 to 1991.25   I said the positions in the data base are as they were in 1991.25, but using the ICRS of 2000.  

With no changes, ephem applies proper motion (so many milli arc seconds per year) assuming the numbers are from 2000.  In other words, they are short by 8.75 years. Done correctly, we should match the usno site.  

If you change 2000 to 1991.25 the code uses the correct interval of years for proper motion, but it also assumes the ICRS is epoch 1991.25, and translates from 1991.25 to epoch of date (but it should translate from J2000 to epoch of date.)

When I made changes to my local copy,  I get
>>> s.compute(ephem.date("2017/1/1 12:00"))
>>> s.ra
14:16:25.07
>>> s.dec
19:05:39.6

which are much close to the usno numbers.  But I am not an expert on this, and leave it to the developers....
